### PR TITLE
Minor simplification of nested ifs

### PR DESCRIPTION
**Pull request by lokal-profil**

### CODE REVIEW

File: wikidataStuff/WikidataStuff.py [hunk 1]
```diff
@@ -243,18 +243,17 @@ def addLabelOrAlias(self, lang, name, item, summary=None,
             pywikibot.output(edit_summary)
         elif name != item.labels[lang]:
             # look at aliases
-            if not caseSensitive:
-                if name.lower() == item.labels[lang].lower():
-                    return None
+            if not caseSensitive and name.lower() == item.labels[lang].lower():
```
Comment (Vesihiisi) [inline wikidataStuff/WikidataStuff.py]: Use parentheses to clarify the order of the logic?

File: wikidataStuff/WikidataStuff.py [hunk 2]
```diff
@@ -243,18 +243,17 @@ def addLabelOrAlias(self, lang, name, item, summary=None,
...
             if not item.aliases or lang not in item.aliases:
                 aliases = {lang: [name, ]}
                 item.editAliases(aliases, summary=edit_summary)
                 pywikibot.output(edit_summary)
             elif name not in item.aliases[lang]:
-                if not caseSensitive:
-                    if name.lower() in list_to_lower(item.aliases[lang]):
-                        return None
+                if not caseSensitive and \
+                        name.lower() in list_to_lower(item.aliases[lang]):
```
Comment (Vesihiisi) [inline wikidataStuff/WikidataStuff.py]: Same as above about clarifying order.

Comment (lokal-profil) [inline wikidataStuff/WikidataStuff.py]: Done

File: wikidataStuff/WikidataStuff.py (hunk 1 above)
Comment (lokal-profil) [inline wikidataStuff/WikidataStuff.py]: Done

**Pull request merged**